sdk/python/feast/data_source.py

```diff
@@ -271,6 +279,13 @@ def __eq__(self, other):
 
         return True
 
+    @property
```

Comment:
btw @felixwang9817 has started to move us to `attributes` instead of having all these property getter/setters. May be worth sneaking in.